src/consensus_utils.erl

```diff
@@ -14,8 +14,8 @@
 %% API functions
 %%====================================================================
 
--spec atom_to_char(atom()) -> char().
-atom_to_char(Atom) -> hd(atom_to_list(Atom)).
+-spec binary_to_char(binary()) -> char().
+binary_to_char(Bin) -> hd(binary_to_list(Bin)).
```

Comment:
hd() will throw an error if the list is empty, it may be this list will never be empty, but is there the possibility of an empty binary here?

Reply:
It's not a useful operation on an empty binary, so I'll probably just leave it. Thanks.